### PR TITLE
Actor - Role - Movie relationship

### PR DESCRIPTION
**Pull request by paskowski**
Implemented simpler relationships between actors and their roles in movies

### CODE REVIEW

File: .vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
```
Comment (paskowski) [inline .vscode/launch.json:1]: I debugged a lot of stuff and VSC had issues with creating proper config so I decided to add it to repo after finally getting it right

File: acf-config.php [hunk 2]
```diff
@@ -17,3 +17,9 @@ function my_acf_settings_url( $url ) {
 function my_acf_settings_show_admin( $show_admin ) {
     return false;
 }
+
+function replace_repeater_field( $where ) {
```
Comment (paskowski) [inline acf-config.php:21]: This filter helps with querying data stored in repeater

File: features/actor/custom-fields.php [hunk 3]
```diff
@@ -23,12 +23,6 @@ function aktorzy_field_groups() {
                 'label' => 'Miejsce urodzenia',
                 'name' => 'place_of_birth',
                 'type' => 'text',
-            ),
```
Comment (paskowski) [inline features/actor/custom-fields.php:26]: Actor will no longer keep info about movies in which he/she plays

File: features/actor/single/template.php [hunk 4]
```diff
@@ -23,21 +23,52 @@
...
+                    'meta_query' => array(
+                        array(
+                            'key' => 'actors_repeater_$_actor',
+                            'value' => $current_actor_id,
+                            'compare' => '='
+                        )
+                    )
+                );
+
+                $loop = new WP_Query($args);
```
Comment (Sikora00) [inline features/actor/single/template.php:42]: Is it posible to define some function like getActorMoviesIterator($id) outside of the template?

**Review by Sikora00 — CHANGES_REQUESTED** [review]

File: features/actor/single/template.php (hunk 4 above)
Comment (paskowski) [reply features/actor/single/template.php:42]: I will look into that

**Review by s17534 — APPROVED** [review]

**Review by Sikora00 — APPROVED** [review]

**Pull request merged**